yamcs-api/src/main/proto/yamcs/protobuf/filetransfer/filetransfer.proto

```diff
@@ -76,6 +77,29 @@ service FileTransferApi {
       topic: "file-transfers"
     };
   }
+
+  // Receive remote file lists
+  rpc SubscribeRemoteFileList(SubscribeTransfersRequest) returns (stream ListFilesResponse) {
+    option (yamcs.api.websocket) = {
+      topic: "remote-file-list"
+    };
+  }
+
+  // Request file list from remote
+  rpc RequestFileList(ListFilesRequest) returns (google.protobuf.Empty) {
+    option (yamcs.api.route) = {
+      post: "/api/filetransfer/{instance}/{serviceName}/requestfiles"
```

Comment:
Verbs in the path should be separated by colon, I suggest maybe this:
`post: "/api/filetransfer/{instance}/{serviceName}/files:sync"`